### PR TITLE
Issue #3042078: Update to Paragraphs 1.9

### PR DESCRIPTION
**Pull request by chrfritsch**
Make sure these boxes are checked before submitting your pull request - thank you!

- [ ] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [ ] All tests are running locally. ([How to run the test?](https://github.com/BurdaMagazinOrg/thunder-distribution/blob/develop/docs/development.md#how-to-run-the-tests))
- [ ] Necessary update hooks are provided.
- [ ] User roles have correct access for new introduced permission.
- [ ] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [ ] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -127,7 +124,7 @@
         "drupal/media_expire": "^1.0",
         "drupal/nexx_integration": "^1.0||^2.0",
         "drupal/metatag": "^1.0",
-        "drupal/paragraphs": "1.6",
+        "drupal/paragraphs": "^1.0",
```
Comment (dbosen) [inline composer.json]: needs to be ^1.7

File: composer.json [hunk 2]
```diff
@@ -62,9 +62,6 @@
             "drupal/diff": {
                 "Back button for comparison page": "https://www.drupal.org/files/issues/back_button_for-2853193-4.patch"
             },
-            "drupal/paragraphs": {
-                "Support of add above functionality in Thunder": "https://www.drupal.org/files/issues/2018-09-28/2977454_7.patch"
```
Comment (dbosen) [inline composer.json:66]: Yes!

**Review by dbosen — DISMISSED** [review]

File: composer.json (hunk 1 above)
Comment (chrfritsch) [reply composer.json]: Done

File: thunder.install [hunk 3]
```diff
@@ -1181,3 +1181,27 @@ function thunder_update_8137() {
  */
 function thunder_update_8138() {
 }
+
+/**
+ * Implements hook_requirements().
+ */
+function thunder_requirements($phase) {
+  if ($phase !== 'update') {
```
Comment (dbosen) [inline thunder.install]: maybe we can use drupal_valid_test_ua() here and crap the patch for tests.

**Review by dbosen — CHANGES_REQUESTED** [review]
see comment

**Review by dbosen — APPROVED** [review]

**Pull request merged**